LayoutTests/svg/crash-svg-filter-empty-viewport.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="0" height="0" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" onload="test()">
+  <filter id="filter"></filter>
+  <marker id="marker" orient="1" filter="url(#filter)"></marker>
+  <line marker-end="url(#marker)"></line>
+  <text>PASS</text>
+  <script>
+    function test() {
+      let filter = document.getElementById("filter");
+      let length = filter.x.baseVal;
+      length.value = 1;
```

Comment:
Should not we verify the value of length after setting it?
```
shouldBe("length.unitType", "SVGAngle.SVG_LENGTHTYPE_PERCENTAGE");
shouldBe("length.value", "1");
```

Reply:
I added that, but because the value is calculated against the empty viewport length.value ends up being zero, matching what Chrome does.